src/ucp/api/ucp.h

```diff
@@ -1564,12 +1559,13 @@ ucs_status_t ucp_worker_fence(ucp_worker_h worker);
 /**
  * @ingroup UCP_WORKER
  *
- * @brief Flush all outstanding communication on the @ref ucp_worker_h "worker"
+ * @brief Flush outstanding one-sided operations on the @ref ucp_worker_h
```

Comment:
Sounds ok to me. We have to update it somehow to make it consistent with ucp_ep_flush description. It already mentions "one-sided" operations. So we may use "AMO and RMA" instead.